gallery/src/styles/NxPageTitle/NxPageTitleEverythingExample.tsx

```diff
@@ -57,7 +57,8 @@ function NxPageTitleEverythingExample() {
         <NxThreatCounter criticalCount={45}
                          severeCount={21114}
                          moderateCount={12}
-                         lowCount={45}/>
+                         lowCount={45}
+                         noneCount={21}/>
```

Comment:
While we do certainly want these examples that have every count shown, I'm wondering if we also want examples of each layout fewer counts shown.  Keep in mind that these examples double as our visual test items, so sometimes it makes sense to show every possibility even if it might be considered excessive for the sake of human documentation.

Reply:
> While we do certainly want these examples that have every count shown, I'm wondering if we also want examples of each layout fewer counts shown. Keep in mind that these examples double as our visual test items, so sometimes it makes sense to show every possibility even if it might be considered excessive for the sake of human documentation.

I've added some more examples to the existing pages https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/eff6a8606c1b78944ee9d99b15de9dabff6510fa https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/e5941cc6215ee25c84289e53ead4335f4b13b760 https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/c7a4d1230c210154b7502dc9699f8e7bdd329f0c) what do you think of these?
![image](https://user-images.githubusercontent.com/6968527/122973907-a5c40780-d389-11eb-91da-ff7f3476eb70.png)
![image](https://user-images.githubusercontent.com/6968527/122973940-aeb4d900-d389-11eb-82f1-ecebea784fa1.png)
![image](https://user-images.githubusercontent.com/6968527/122973980-b83e4100-d389-11eb-80e3-c461a738502f.png)

Reply:
> I've added some more examples to the existing pages eff6a86 e5941cc c7a4d12 1915f11 what do you think of these?

Hmm unfortunately I think having them displayed like that makes it difficult for the user to tell what they're looking at, e.g. that in the first one that is 5 separate components laid out one after another vertically.  I think for each layout type, two distinct examples are warranted: one with all the counters and one with less than all of them.

Reply:
> Hmm unfortunately I think having them displayed like that makes it difficult for the user to tell what they're looking at, e.g. that in the first one that is 5 separate components laid out one after another vertically. I think for each layout type, two distinct examples are warranted: one with all the counters and one with less than all of them.

Done in https://github.com/sonatype/sonatype-react-shared-components/pull/418/commits/c296dfb05ada74f6812ddba6a75396b9945f6d76 (each layout with an example of all counts and just 2 counts)